packages/portal-components/component-example.scss

```
@@ -59,6 +59,16 @@
         margin-top: $layout-spacing--small;
     }
 
+    @include small-device {
+        width: calc(100% - #{$layout-spacing--medium});
+        max-width: 100%;
+        min-width: initial;
+        flex-direction: column;
+        &__example-wrapper {
+            width: initial;
+        }
```

```suggestion
        &__example-wrapper {
            width: initial;
        }
        &__example-options {
            padding: 0;
            padding-top: $component-spacing--medium;
        }
```

For å bli kvitt ekstra padding rundt valgene